### PR TITLE
Added delta to bar chart in hover tooltip for Project Performance Dashboard

### PR DESCRIPTION
**Pull request by rickypchen**
@esoergel @czue Per spec on [Google Doc](https://docs.google.com/spreadsheets/d/1hr5-u9Ni90yhumGJqvROeNvqqn0daIPRVLuzCiIbwb8/edit?ts=5760652c#gid=1814269798), user can hover over bar graph to see changes in number of high/low performing users

screen shot 2016-07-18 at 4 15 58 pm

### CODE REVIEW

File: corehq/apps/reports/standard/project_health.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ def __init__(self, domain, month, users, has_filter, performance_threshold, prev
             total_users_by_month=0,
             percent_active=0,
             performing=num_performing_user,
+            delta_num_high_performing_users=0,
+            delta_num_low_performing_users=0,
```
Comment (esoergel) [inline corehq/apps/reports/standard/project_health.py]: Any reason these two aren't just passed in to the constructor for this class as well?

**Comment by esoergel** [comment]
This one also has conflicts.

**Comment by czue** [comment]
nice little change. would be nice if it used the little chevrons in the other deltas in the report if that's not too hard. then I think the text could be cleaner if it just said "up/down [number] from last month" where up and down could optionally be a chevron

**Comment by czue** [comment]
didn't have any other feedback apart from ethan's

**Comment by rickypchen** [comment]
Resolved merge conflict. @czue @esoergel Looking forward to getting feedback on the refactoring for setting delta_high_performers and delta_low_performers inside the constructor.

ps. Added chevron to tooltip per @czue's suggestion
screen shot 2016-07-19 at 2 39 14 pm

**Pull request merged**